test/src/test/java/io/pravega/schemaregistry/integrationtest/TestPravegaClientEndToEnd.java

```diff
@@ -1377,73 +1372,5 @@ public void testWithSchema() {
         }
     }
 
-    @Test
-    public void testKeyValue() {
```

Comment:
Why are we removing a test as part of pravega client upgrade?

Reply:
This test was removed as per Andrei's suggestion that either we fix the code to use Client KVT or we remove the code which creates KVT from Schema Registry (including tests). This test will be included again with newer implementation with Client KVT